superset/connectors/sqla/models.py

```diff
@@ -924,9 +934,15 @@ def get_sqla_query(  # sqla
                     elif op == utils.FilterOperator.LIKE.value:
                         where_clause_and.append(col_obj.get_sqla_col().like(eq))
                     elif op == utils.FilterOperator.IS_NULL.value:
-                        where_clause_and.append(col_obj.get_sqla_col() == None)
+                        where_clause_and.append(
+                            col_obj.get_sqla_col()  # pylint: disable=singleton-comparison
```

Comment:
I think `is None` and `is not None` will resolve lines 938 and 943 respectively.

Reply:
You know, I thought about that, even tried it, and I'm pretty sure that'll break the function:

```python
>>> from sqlalchemy.sql import column
>>> column("test") is not None
True
>>> column("test") is None
False
>>> column("test") == None
<sqlalchemy.sql.elements.BinaryExpression object at 0x108445e48>
>>> column("test") != None
<sqlalchemy.sql.elements.BinaryExpression object at 0x108450668>
```
I believe this is building up a `where` clause, not returning booleans.